Controllers/UserController.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Net;
+
+namespace UserLookupService.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class UserController : ControllerBase
+{
+    
+    private readonly ILogger<UserController> _logger;
+
+    public UserController(ILogger<UserController> logger)
+    {
+        _logger = logger;
+    }
+
+    [HttpGet("")]
+    public IActionResult Get()
+    {
+        _logger.LogInformation("We hit our UserController");
```

Comment:
We should remove the line of logging, we don't need it anymore now that we know it's working. 
```suggestion

```